### ISSUE TITLE
About Tokoy dataset

### ISSUE THREAD

**Issue by DonMuv** [QVPR/Patch-NetVLAD#40]
HI,
When doing experiments on the Tokoy dataset, TimeMachine is used for the training set, Tokoy24/7 is used for the test set, is that right?

**Comment by StephenHausler** [comment]
Hi @DonMuv, we resize all our images to a uniform size of 640x480 (width by height), including Tokyo247 query images. This resizing happens as part of the Pytorch transforms part of the dataloader.
